rclc/src/rclc/executor.c

```diff
@@ -958,6 +1366,41 @@ _rclc_take_new_data(rclc_executor_handle_t * handle, rcl_wait_set_t * wait_set)
   return rc;
 }
 
+bool _rclc_check_handle_data_available(rclc_executor_handle_t * handle)
+{
+  switch (handle->type) {
+    case ACTION_CLIENT:
+      if (handle->action_client->feedback_available == true ||
```

Comment:
Are the explicit `== true` necessary?

Reply:
fc96136823bd5844fecebcd63ddc8788c031fcb4